.github/workflows/release.yml

```diff
@@ -0,0 +1,72 @@
+name: release
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*.*.*
+  workflow_dispatch:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            target: x86_64-unknown-linux-gnu
+            suffix: ""
+          - os: macos-latest
+            target: x86_64-apple-darwin
+            suffix: ""
+          - os: macos-latest
+            target: aarch64-apple-darwin
+            suffix: ""
+          - os: windows-latest
+            target: x86_64-pc-windows-msvc
+            suffix: ".exe"
```

Comment:
the uploaded files for mac and linux should be compressed

Reply:
You mean using `gzip -c svd2rust > svd2rust-<target>.gz` and upload that?